legal-api/Makefile

```diff
@@ -133,9 +133,6 @@ tag: push ## tag image
 run: ## Run the project in local
 	. venv/bin/activate && python -m flask run -p 5050
```

Comment:
can we revert this to port 5000?

Reply:
Sure can. 5000 sometimes overlaps with Apples AirPlay server and things can get weird.